docs/topics/documenting-your-api.md

```diff
@@ -4,176 +4,121 @@
-* `public`: Default `True`. If `True` schema is generated without a `request` instance being passed to views.
-* `patterns`: Default `None`. A list of URLs to inspect when generating the schema. If `None` project's URL conf will be used.
-* `generator_class`: Default `rest_framework.schemas.SchemaGenerator`. May be used to specify a `SchemaGenerator` subclass to be passed to the `SchemaView`.
-* `authentication_classes`: Default `api_settings.DEFAULT_AUTHENTICATION_CLASSES`. May be used to pass custom authentication classes to the `SchemaView`.
-* `permission_classes`: Default `api_settings.DEFAULT_PERMISSION_CLASSES`. May be used to pass custom permission classes to the `SchemaView`.
-* `renderer_classes`: Default `None`. May be used to pass custom renderer classes to the `SchemaView`. If `None` the `SchemaView` will be configured with `DocumentationRenderer` and `CoreJSONRenderer` renderers, corresponding to the (default) `html` and `corejson` formats.
-
-#### `get_schemajs_view`
-
-* `title`: Default `None`. May be used to provide a descriptive title for the schema definition.
-* `description`: Default `None`. May be used to provide a description for the schema definition.
-* `schema_url`: Default `None`. May be used to pass a canonical base URL for the schema.
-* `public`: Default `True`. If `True` schema is generated without a `request` instance being passed to views.
-* `patterns`: Default `None`. A list of URLs to inspect when generating the schema. If `None` project's URL conf will be used.
-* `generator_class`: Default `rest_framework.schemas.SchemaGenerator`. May be used to specify a `SchemaGenerator` subclass to be passed to the `SchemaView`.
-* `authentication_classes`: Default `api_settings.DEFAULT_AUTHENTICATION_CLASSES`. May be used to pass custom authentication classes to the `SchemaView`.
-* `permission_classes`: Default `api_settings.DEFAULT_PERMISSION_CLASSES` May be used to pass custom permission classes to the `SchemaView`.
-
-
-### Customising code samples
-
-The built-in API documentation includes automatically generated code samples for
-each of the available API client libraries.
-
-You may customise these samples by subclassing `DocumentationRenderer`, setting
-`languages` to the list of languages you wish to support:
-
-    from rest_framework.renderers import DocumentationRenderer
-
-
-    class CustomRenderer(DocumentationRenderer):
-        languages = ['ruby', 'go']
-
-For each language you need to provide an `intro` template, detailing installation instructions and such,
-plus a generic template for making API requests, that can be filled with individual request details.
-See the [templates for the bundled languages][client-library-templates] for examples.
-
----
+REST framework provides built-in support for generating OpenAPI schemas, which
+can be used with tools that allow you to build API documentation.
+
+There are also a number of great third-party documentation packages available.
```

Comment:
Hey @tomchristie. Can you review the Third party section on this page at 1d3a3ee3604fb6eb3132bc00aaf93c0dcba2c639? 

I ≈think we could just drop it... 

* drf-yasg is already mentioned on the Schema page, and it's not about Docs. 
* Not sure if DRF AutoDocs and/or Django REST Swagger are still active. 
* Is Apiary still worth calling out (from all the commercial solutions available now)? 

Seems like a reasonable chance to review. Thanks!

Reply:
Ref @rpkilby's #6733

Reply:
Just a followup... #7057
- drf-autodocs looks unmaintained. No meaningful commits since ~2017, and it doesn't support Django 2.0
- Removed Apiary because agreed, why call out one specific commercial solution?